x/mint/keeper/keeper.go

```diff
@@ -137,56 +143,99 @@ func (k Keeper) GetProportions(ctx sdk.Context, mintedCoin sdk.Coin, ratio sdk.D
 	return sdk.NewCoin(mintedCoin.Denom, mintedCoin.Amount.ToDec().Mul(ratio).TruncateInt())
 }
 
+const (
+	// TODO (set this!)
+	// strategic reserve address
+	StrategicReserveAddress = "stride1vytsydmkkmkndlrkeqkqad7zkx4gpgmn50vmjn"
+)
+
 // DistributeMintedCoins implements distribution of minted coins from mint to external modules.
 func (k Keeper) DistributeMintedCoin(ctx sdk.Context, mintedCoin sdk.Coin) error {
 	params := k.GetParams(ctx)
 	proportions := params.DistributionProportions
 
-	// allocate staking incentives into fee collector account to be moved to on next begin blocker by staking module
-	stakingIncentivesCoins := sdk.NewCoins(k.GetProportions(ctx, mintedCoin, proportions.Staking))
+	k.Logger(ctx).Info(fmt.Sprintf("Distributing minted x/mint rewards: %d coins...", mintedCoin.Amount.Int64()))
+
+	// // allocate staking incentives into fee collector account to be moved to on next begin blocker by staking module
+	stakingIncentivesProportions := k.GetProportions(ctx, mintedCoin, proportions.Staking)
+	stakingIncentivesCoins := sdk.NewCoins(stakingIncentivesProportions)
+	k.Logger(ctx).Info(fmt.Sprintf("\t\t\t...staking rewards: %d to %s", stakingIncentivesProportions.Amount.Int64(), k.feeCollectorName))
```

Comment:
Just making sure I'm following:
Is the idea that the staking rewards go to the fee collector so they can be distributed to all the validators?
Whereas with the other reserves, we just send it straight to an address?

Reply:
precisely!